### DISCUSSION TITLE
能添加Jackett

### DISCUSSION THREAD

**Discussion by zhangsun5151** [jxxghp/MoviePilot#2643] [建议]
能添加Jackett，这个站点比较多

**Comment by zhangsun5151** [comment]
这个项目[https://github.com/Jackett/Jackett](url)

**Comment by MoogleXX** [comment]
Nastool时候就支持的。不知道怎么现在插件里面不支持这个了。

**Comment by chenmdg** [comment]
这个主要是给pt站玩的，bt质量层次不齐。想办法多申请些pt站吧

**Reply by zhangsun5151** [reply]
我是想表达的是，这个插件支持的pt站多好添加？
